package.json

```diff
@@ -45,6 +45,7 @@
     "react-motion": "^0.4.4",
     "react-onclickoutside": "^5.3.3",
     "react-overlays": "^0.6.6",
+    "react-responsive-modal": "^3.5.1",
```

Comment:
Good usage of this! I'm going to look into re-using this for the Login modal too 👍